### DISCUSSION TITLE
启动失败：线程从等待状态中断

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#3806] [讨论]
鉴于无法确认是不是真的 Bug，就扔到讨论里来了
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/fdd44bab-3e19-4a00-abc5-17621cebdf3c)
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/15280661/Log1.txt)

**Comment by wuliaodexiaoluo** [comment]
我选择直接交个 Issue 让龙猫看看

**Reply by wuliaodexiaoluo** [reply]
#3807

**Reply by allMagicNB** [reply]
6
